src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a person's contact in the address book.
+ */
+public abstract class Contact {
```

Comment:
By defining `Contact` as an abstract class, you won't be able to create an instance of type `Contact`.